config.lua

```diff
@@ -7,8 +7,17 @@ Config.LockNPCParkedCars = true -- Lock state for NPC parked cars [true = locked
 -- Lockpick Settings
 Config.RemoveLockpickNormal = 0.5 -- Chance to remove lockpick on fail
 Config.RemoveLockpickAdvanced = 0.2 -- Chance to remove advanced lockpick on fail
-Config.LockPickDoorEvent = function() -- This function is called when a player attempts to lock pick a vehicle
-    TriggerEvent('qb-lockpick:client:openLockpick', LockpickFinishCallback)
+Config.LockPickDoorEvent = function()
+    loadAnimDict('veh@break_in@0h@p_m_one@')
+    TaskPlayAnim(PlayerPedId(), 'veh@break_in@0h@p_m_one@', "low_force_entry_ds", 3.0, 3.0, -1, 16, 0, 0, 0, 0)
+    local success = lib.skillCheck({'easy', 'easy', {areaSize = 60, speedMultiplier = 1}, 'medium'}, {'1', '2', '3', '4'})
+    if success then
+        LockpickFinishCallback(success)
```

Comment:
I think this function would be better if the callback was passed in as an argument, or maybe instead of a callback, the function should trigger an event?

Reply:
i will do that

Reply:
ys

Reply:
> I think this function would be better if the callback was passed in as an argument, or maybe instead of a callback, the function should trigger an event?

i think i should remove this from config and put it in client with triggerevent whats ur thought?

Reply:
Yes that makes sense.

Reply:
> Yes that makes sense.

for now i just removed and put it on client but i will do another pr when i will arrive home